dvc/command/config.py

```diff
@@ -71,7 +72,9 @@ def _list(self):
             prefix = self._config_file_prefix(
                 self.args.show_origin, self.config, level
             )
-            logger.info("\n".join(self._format_config(conf, prefix)))
+            configs = list(self._format_config(conf, prefix))
+            if configs:
+                ui.write("\n".join(configs))
```

Comment:
`conf` might turn out to be empty. Before we were just printing empty lines for some levels. It was easier to fix in the PR itself.